src/psyclone/tests/psyir/nodes/dynamic_omp_task_directive_test.py

```diff
@@ -2961,7 +3047,8 @@ def test_omp_task_external_constant(fortran_reader, fortran_writer, tmpdir):
     enddo
     !$omp end task'''
     assert correct in fortran_writer(psyir)
-    assert Compile(tmpdir).string_compiles(fortran_writer(tree))
+    # Cannot test compilation of this test as it uses an external module.
+    # assert Compile(tmpdir).string_compiles(fortran_writer(tree))
```

Comment:
Please rm this commented-out assert as there's no likelihood we'll ever be able to compile it.

Reply:
Done.